src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -1005,11 +1005,11 @@ do(Function, Args) ->
 -endif.
 
 %% @private
-log_transmission(Term) ->
+log_transmission(Type, Payload, PeerCount) ->
     try
         case lasp_config:get(instrumentation, false) of
             true ->
-                ok = lasp_transmission_instrumentation:log(server, Term, node()),
+                ok = lasp_transmission_instrumentation:log(server, Type, Payload, PeerCount, node()),
```

Comment:
That distinction was so it could run twice on the same server and create two different log files because they each used a different way of tracking transmission.  This is no longer needed, given each instance running during the evaluation has a different purpose.  I'd just remove it, as I think it's no longer needed.